infrastructure/envs/dev/variables.tf

```diff
@@ -6,6 +6,10 @@ variable "tier" {
   default = "dev"
 }
 
+variable "multi_az" {
+  default = false
+}
```

Comment:
Define this variable under `modules/migrations/variables.tf` instead of here, because it's specific to the migration module.